unmarshaler_test.go

```diff
@@ -1171,11 +1238,15 @@ key3 = "value3"
 	err := d.Decode(&s)
 
 	fmt.Println(err.Error())
-	// Output: strict mode: fields in the document are missing in the target struct
 
-	details := err.(*toml.StrictMissingError)
+	var details *toml.StrictMissingError
+	if !errors.As(err, &details) {
+		panic(fmt.Sprintf("err should have been a *toml.StrictMissingError, but got %s (%T)", err, err))
+	}
+
 	fmt.Println(details.String())
-	// Ouput:
+	// Output:
```

Comment:
Spelling mistake was preventing the complete test

Reply:
🤦🏻‍♂️ good catch